forestplot.js

```diff
@@ -210,8 +210,19 @@ function forestplot(data, element, groups, pairs){
         let table = $('.forestplot table').DataTable({ 
             "dom": '<"top"if>rt<"clear">',
             "paging": false, 
-            "order": [[2, "desc"]]
-        });
+            "order": [[2, "desc"]],
+            "columns": [
+                { "width": "2%" },
```

Comment:
One other note: Column counts are subject to change, so this would likely need need to be generated dynamically with percentages derived based on the `groups` and  `diffs` parameters.